octopusCatsEffect/src/main/scala/octopus/async/cats/cats.scala

```diff
@@ -0,0 +1,25 @@
+package octopus.async
+
+import _root_.cats.effect.IO
+import _root_.cats.implicits._
+import _root_.cats.Applicative
+
+import octopus.AppError
+
+package object cats {
+
+  implicit val catsIOAppError: AppError[IO] = new AppError[IO] {
```

Comment:
I think this typeclass could be implemented for any `[F[_]](implicit F: ApplicativeError[F, Throwable])` (or maybe monaderror)

Reply:
That's true, I was able to implement it only in terms of ApplicativeError for cats and MonadError for scalaz (which I think misses ApplicativeError). This is big benefit, as we would not need to provide additional ***-effect related modules.